TheWarWithin/ShamanRestoration.lua

```diff
@@ -1038,13 +1050,26 @@ spec:RegisterAbilities( {
 } )
 
 
-spec:RegisterSetting( "experimental_msg", nil, {
+--[[spec:RegisterSetting( "experimental_msg", nil, {
     type = "description",
     name = "|cFFFF0000WARNING|r:  Healer support in this addon is focused on DPS output only.  This is more useful for solo content or downtime when your healing output is less critical in a group/encounter.  Use at your own risk.",
     width = "full",
+} )--]]
+
+spec:RegisterSetting( "experimental_msg", nil, {
+    type = "description",
+    name = "Restoration Shaman currently has support for some basic healing maintenance with the totemic chain heal build. It will not necessarily tell you how to heal, but will assist with keeping healing rain / surging totem up, stop you from overcapping on riptide and healing stream totem, use unleash life on CD with good spells and maintain your earth shields.",
+    width = "full",
 } )
 
---[[spec:RegisterSetting( "second_shield", "earth_shield", {
+spec:RegisterSetting( "healing_mode", false, {
+    name = "Healing Helper Mode",
+    desc = "Restoration Shaman currently has support for some basic healing maintenance with the totemic chain heal build. It will not necessarily tell you how to heal, but will assist with keeping healing rain / surging totem up, stop you from overcapping on riptide and healing stream totem, use unleash life on CD with good spells and maintain your earth shields.",
```

Comment:
Avoid duplicative description; if it's spelled out above, this can be abbreviated.